server/src/main/java/org/elasticsearch/action/admin/cluster/node/tasks/get/GetTaskAction.java

```diff
@@ -25,6 +25,7 @@
  * Action for retrieving a list of currently running tasks
  */
 public class GetTaskAction extends Action<GetTaskResponse> {
+    public static final String TASKS_ORIGIN = "tasks";
```

Comment:
I stuck this here because it felt like a fairly right place at the time. Now that I look at it again maybe it should be on something a little more centrally located. But it is *about* actions so I stuck it on an action. I'm not sure of a better spot.